.github/workflows/all-nodejs-packages-publish.yaml

```diff
@@ -4,9 +4,12 @@ env:
   NODEJS_VERSION: v18.18.2
 
 on:
-  push:
+  pull_request:
+    branches: 
+      - main
+      - dev
 
```

Comment:
@raynatopedrajeta Wouldn't adding this make it so that this workflow gets executed for every pull request instead of just the tags starting with the character `v`? 
It's been a while since I've dealt with these configurations so I'm not sure, but it looks like it would be doing the opposite of the task, e.g. instead of reducing the resource usage it would increase it.